webapp-django/crashstats/crashstats/jinja2/crashstats/bugzilla_comment.txt

```diff
@@ -0,0 +1,18 @@
+{# This is a Jinja2 template for Bugzilla bugs filed from the Socorro interface.
+   See crashstats.templatetags.jinja_helpers.bugzilla_submit_url. #}
+{%- macro frame_source(frame) -%}
+  {{ frame.file if frame.file }}{{ ':%s'|format(frame.line) if frame.line }}
+{%- endmacro -%}
+
+This bug was filed from the Socorro interface and is
+report bp-{{ uuid }}.
+=============================================================
+
+{% if parsed_dump.threads and crashing_thread is not none -%}
+Top 10 frames of crashing thread:
```

Comment:
If the stack has fewer than 10 frames, would someone write up a bug for Socorro because this number doesn't match the number of frames we generated?